dts/arm/gigadevice/gd32f4xx/gd32f4xx.dtsi

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2021, Teslabs Engineering S.L.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv7-m.dtsi>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu0: cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m4f";
+			reg = <0>;
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			mpu: mpu@e000ed90 {
+				compatible = "arm,armv7m-mpu";
+				reg = <0xe000ed90 0x40>;
+				arm,num-mpu-regions = <8>;
+			};
+		};
+	};
+
+	soc {
+		sram0: memory@20000000 {
+			compatible = "mmio-sram";
+		};
```

Comment:
SRAM0, SRAM1, DTCM and BKP SRAM are same for all F4XX series and should be define here

Reply:
Moved sram0 to gd32f4xx.dtsi file, other rams have not defined (some are not supported yet, others likely have specific purpose again unsupported)